### PR TITLE
Add Spanish random names API

### PR DESCRIPTION
**Pull request by leahfern**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

NOTE: This is a re-submission of inactive PR #1962 that fulfills the request to clean up the edits to only the API being added. Also made a minor correction to the description. #1962 can now be closed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1104,6 +1104,7 @@ API | Description | Auth | HTTPS | CORS |
 | [RandomUser](https://randomuser.me) | Generates and list user data | No | Yes | Unknown |
 | [RoboHash](https://robohash.org/) | Generate random robot/alien avatars | No | Yes | Unknown |
 | [Spanish random words](https://palabras-aleatorias-public-api.herokuapp.com) | Generate spanish words randomly | No | Yes | Unknown |
+| [Spanish random names](https://random-names-api.herokuapp.com) | Generate spanish names (with gender) randomly | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1107]: Fix the alphabetical order (before `Spanish random words`) and update to correct link, please:

```suggestion
| [Spanish random names](https://random-names-api.herokuapp.com/public) | Generate spanish names (with gender) randomly | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @leahfern 😆 

Can you resolve this review, please?

**Comment by leahfern** [comment]
Hi @matheusfelipeog I've updated the PR with your suggestions - thank you!

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for your update @leahfern ;)

**Comment by matheusfelipeog** [comment]
Ref: #2002

**Pull request merged**